doc-content/enterprise-only/springboot/bus-app-rh-sso_proc.adoc

```diff
@@ -0,0 +1,109 @@
+[id='bus-app-rh-sso_{context}']
+= Configuring the business application with Red Hat Single Sign-On
+
+You can use Red Hat Single Sign-On (RH SSO) to enable single sign-on between your services and to have a central place to configure and manage your users and roles. 
+
+.Prerequisites
+* You have a `<business-application>.zip` file that you created using the http://start.jbpm.org[business applications] web site.
+
+.Procedure
+. Download and install RH SSO. For instructions, see the https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.2/html/getting_started_guide/[_Red Hat Single Sign-On Getting Started Guide_].
+. Configure RH SSO:
+.. Either use the default master realm or create a new realm.
+.. Create the `springboot-app` client and set the `AccessType` to public.
+.. Set a valid redirect URI and web origin according to your local setup, for example:
++
+* Valid redirect URIs: http://localhost:8090/*
+* Web Origins: http://localhost:8090
+.. Create realm roles that are used in the application.
+.. Create users used in the application and assign roles to them.
+. Add the following dependencies to service project `pom.xml` file:
++
+[source, xml]
+----
+<dependencyManagement>
```

Comment:
@MarianMacik I think I over generalized changing keycloak to rhsso in the examples - I checked the PAM/DM RH-SSO docs. Please confirm this is now correct.